src/topology/algebra/mul_action.lean

```diff
@@ -58,6 +58,11 @@ lemma filter.tendsto.const_smul {f : β → α} {l : filter β} {a : α} (hf : t
   tendsto (λ x, c • f x) l (𝓝 (c • a)) :=
 tendsto_const_nhds.smul hf
 
+lemma filter.tendsto.smul_const {f : β → M} {l : filter β} {c : M}
+  (hf : tendsto f l (𝓝 c)) (a : α) :
+  tendsto (λ x, (f x) • a) l (𝓝 (c • a)) :=
+hf.smul tendsto_const_nhds
+
 variables [topological_space β] {f : β → M} {g : β → α} {b : β} {s : set β}
 
```

Comment:
There's a whole collection of `.const_smul` lemmas below this line - perhaps those should also be duplicated for `smul_const`?

Reply:
It sounds sensible. Maybe there are even more of those around in mathlib?